validphys2/src/validphys/tableloader.py

```diff
@@ -41,6 +41,11 @@ def parse_data_cv(filename):
     df = sane_load(filename, index_col=[0, 1, 2])
     return df
 
+def parse_chi2_data(filename):
+    """Useful for reading whole DataFrames."""
+    df = sane_load(filename)
+    return df
+
```

Comment:
You _should_ not need that function, it _should_ be possible to use directly `sane_load`...

Unless the object produced by `partial` and `def` are different for whatever is important for the decorator below, they are basically the same function...

Reply:
Yes, you are right [b9cc5b3](https://github.com/NNPDF/nnpdf/pull/1910/commits/b9cc5b3466be3bb5139c284ce5d47482a1b17515)